crates/egui_extras/src/layout.rs

```diff
@@ -101,7 +101,13 @@ impl<'l> StripLayout<'l> {
         let rect = self.cell_rect(&width, &height);
         let used_rect = self.cell(rect, add_contents);
         self.set_pos(rect);
-        self.ui.allocate_rect(rect.union(used_rect), Sense::hover())
+        let rect = if !rect.contains_rect(used_rect) {
+            rect.union(self.ui.available_rect_before_wrap())
+        } else {
+            used_rect
+        };
```

Comment:
perhaps you could add a comment here explaining the logic of this

Reply:
I changed that line to `self.ui.available_rect_before_wrap()` and added a comment.